.github/workflows/stale_pull_requests.yml

```diff
@@ -0,0 +1,20 @@
+name: 'Close stale pull requests'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+  workflow_dispatch:
+
+jobs:
+  stale:
+    if: ${{ github.repository_owner == 'pytorch' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3
+        with:
+          stale-pr-message: >
+            Looks like this PR hasn't been updated in a while! Going to go ahead and mark this as `stale`.
+            Feel free to update / remove the `stale` label if you feel this is a mistake
+            `stale` pull requests will automatically be closed 30 days after being marked `stale`
+          exempt-pr-labels: "no-stale"
```

Comment:
If we add "open source" and "high priority" exemptions, then I think we can get more aggressive and close after as little as 3 months.

Reply:
I've added a separate job specifically for open source pull requests only